GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/Skorvald.cs

```diff
@@ -590,15 +533,60 @@ internal override void ComputeEnvironmentCombatReplayDecorations(ParsedEvtcLog l
             base.ComputeEnvironmentCombatReplayDecorations(log);
 
             // Mist Bomb - Both for Skorvald and Flux Anomalies
-            if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.MistBomb, out IReadOnlyList<EffectEvent> mistBombEffects))
+            if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.MistBomb, out IReadOnlyList<EffectEvent> mistBombs))
+            {
+                foreach (EffectEvent effect in mistBombs)
+                {
+                    (long start, long end) lifespan = effect.ComputeLifespan(log, 1000);
+                    EnvironmentDecorations.Add(new CircleDecoration(130, lifespan, Colors.Orange, 0.2, new PositionConnector(effect.Position)));
+                }
+            }
+
+            // Solar Bolt - Indicator
+            if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.SolarBoltIndicators, out IReadOnlyList<EffectEvent> solarBoltIndicators))
             {
-                foreach (EffectEvent mistBombEffect in mistBombEffects)
+                foreach (EffectEvent effect in solarBoltIndicators)
```

Comment:
Same code as in Ensolyss and Siax.

Reply:
should i call it "AddCausticBarrageOrSolarBoltDecoration" or got something better?

Reply:
AddDistanceCorrectedOrbDecorations?

Reply:
(ParsedEvtcLog log, int targetID, double distanceThreshold, double onDistanceSuccessDuration, double onDistanceFailDuration)

Reply:
Do you want it located inside fractal logic or combat replay?

Reply:
FractalLogic, at least for now